pkg/descheduler/descheduler.go

```diff
@@ -386,8 +389,10 @@ func RunDeschedulerStrategies(ctx context.Context, rs *options.DeschedulerServer
 			// handle or function which the Evictor has access to. For migration/in-progress framework
 			// work, we are currently passing this via context. To be removed
 			// (See discussion thread https://github.com/kubernetes-sigs/descheduler/pull/885#discussion_r919962292)
+			strategyStart := time.Now()
 			childCtx := context.WithValue(ctx, "strategyName", pg.Name())
 			status := pg.Deschedule(childCtx, nodes)
+			metrics.DeschedulerStrategyDuration.With(map[string]string{"strategy": pg.Name()}).Observe(time.Since(strategyStart).Seconds())
```

Comment:
if multiple profiles use the same strategy, then we will have multiple entries for it. I think that's still ok given that we run them all sequentially.

Reply:
Actually, good that you brought this up. I think it is better to track the profile along with strategy so that it can be a bit useful in the form of metrics. But I am not sure how drastic the cardinality growth might be,